tests/acceptance/features/apiSharingNgShareInvitation/updateShareInvitations.feature

```diff
@@ -1427,3 +1427,358 @@ Feature: Update permission of a share
       | Space Viewer     |
       | Space Editor     |
       | Manager          |
+
+  @env-config
+  Scenario Outline: update share role of file to an existing role after assigned share role is disabled (Personal Space)
+    Given the administrator has enabled the permissions role "Secure Viewer"
+    And user "Alice" has uploaded file with content "hello world" to "textfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt  |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Secure Viewer |
+    And the administrator has disabled the permissions role "Secure Viewer"
+    When user "Alice" updates the last resource share with the following properties using the Graph API:
+      | permissionsRole | <new-permissions-role> |
+      | space           | Personal               |
+      | resource        | textfile.txt           |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "grantedToV2",
+          "id",
+          "roles"
+        ],
+        "properties": {
+          "grantedToV2": {
+            "type": "object",
+            "required": ["user"],
+            "properties":{
+              "user": {
+                "type": "object",
+                "required": [
+                  "displayName",
+                  "id"
+                ],
+                "properties": {
+                  "displayName": {
+                    "const": "Brian Murphy"
+                  },
+                  "id": {
+                    "pattern": "^%user_id_pattern%$"
+                  }
+                }
+              }
+            }
+          },
+          "id": {
+            "pattern": "^%permissions_id_pattern%$"
+          },
+          "roles": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 1,
+            "items": {
+              "pattern": "^%role_id_pattern%$"
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | new-permissions-role |
+      | File Editor          |
+      | Viewer               |
+
+  @env-config
+  Scenario Outline: update share role of folder to an existing role after assigned share role is disabled (Personal Space)
+    Given the administrator has enabled the following permissions roles:
+      | Secure Viewer |
+      | Denied        |
```

Comment:
why two role has been enabled same for below scenario for project space

Reply:
Enabling both the roles at same time and later can update the permissions role to one of the role as both are disabled by default.

One of the test case will be:
1. enable the additional roles
2. share resource with permission role `Secure Viewer`
3. disable the permission role `Secure Viewer`
4. update the share permission role to `Denied`

For 4th step we need to have the `Denied` permissions role to be enabled. So, enabling both the permissions role at same time.